CRM/Contribute/Form/Search.php

```diff
@@ -390,6 +390,11 @@ public function fixFormValues() {
       $this->_defaults['contribution_status_id'] = array($status => 1);
     }
 
+    $pcpid=CRM_Utils_Request::retrieve('pcpid', 'Positive', $this);
+    if ($pcpid) {
```

Comment:
Which input format are we expecting?
I assumed the caller was just adding pcpid=<id> to the query string.
I see you changed the type from 'Positive' to 'String'. Does this mean that we can pass an url parameter like pcid=3,4,5 ? Is the return type from CRM_Utils_Request::retrieve('pcpid', 'String', $this) always an array?